js/models/messages.js

```diff
@@ -1253,32 +1253,67 @@
       // if we did not sync or trigger a sync message for this specific message already
       const shouldTriggerSyncMessage =
         !isOurDevice &&
-        !isOpenGroupMessage &&
```

Comment:
This doesn't matter right now as we're currently not doing any syncing, but in principle it's correct that open group messages aren't synced

Reply:
I removed it from here, because the whole sync block is now on the `if` excluding OpenGroupMessage at https://github.com/loki-project/session-desktop/pull/1346/files/3a876c7322c97bde8df9aaf74e3a3e067c781d8a#diff-fdb432d7888e10d18d02c7becd34c5e4R1309